ablySpec/RealtimeClient.connection.swift

```diff
@@ -589,6 +589,80 @@ class RealtimeClientConnection: QuickSpec {
                     
                 }
 
+                // RTN7b
+                context("ProtocolMessage") {
+
+                    class TotalMessages {
+                        static var expected: Int32 = 0
+                        static var succeeded: Int32 = 0
+                        private init() {}
+                    }
+
+                    it("should contain unique serially incrementing msgSerial along with the count") {
+                        client.connect()
+                        defer {
+                            client.dispose()
+                            client.close()
+                        }
+
+                        let channel = client.channel("channel")
+
+                        waitUntil(timeout: testTimeout) { done in
+                            channel.subscribeToStateChanges { state, status in
```

Comment:
Is all this subscribe prologue necessary? Can't you just publish to the channel so that the message is queue?